README.md

```diff
@@ -62,6 +104,17 @@ Finally lets start the built in Django web server and make sure you can access y
 
 You are now ready to take what you have learned from the Django Treehouse modules and convert the FEWD content to be served via the Django backend.
 
+**Advanced project layout**
```

Comment:
This template has _a lot_ of stuff in it, and it's a pretty different layout than you get from `startapp`. There are some immediate advantages, even for novice Django developers, such as environment settings files, but it's not something a new Python/Django developer will easily grok. Since we have a wide range of skill levels, I wanted to put something in about this. It may make sense for some people to use, and it will give them a sense of how a team's Django project may be set up.